src/main/java/org/scalameter/japi/annotation/ctx.java

```diff
@@ -0,0 +1,17 @@
+package org.scalameter.japi.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/** Optional annotation that points context for a benchmark snippet.
```

Comment:
Optional annotation that points to the context of the annotated benchmarks snippet.